### PR TITLE
PHIOS-6362: Restored EntityAppearanceColor and VehicleAppearanceStyle

### PR DESCRIPTION
**Pull request by stepanrazin**
Related to:
https://1mojio.atlassian.net/browse/PHIOS-6329
https://1mojio.atlassian.net/browse/PHIOS-6362

* Restored EntityAppearanceColor and VehicleAppearanceStyle structures;

### CODE REVIEW

File: MojioSDK/Models/Vehicles/VehicleAppearance.swift [hunk 1]
```diff
@@ -61,7 +67,8 @@ public struct VehicleAppearance: Codable {
 
 extension VehicleAppearance: Equatable {
     public static func == (lhs: VehicleAppearance, rhs: VehicleAppearance) -> Bool {
-        return lhs.vehicleId == rhs.vehicleId && lhs.vehicleImage == rhs.vehicleImage
+        return lhs.vehicleId == rhs.vehicleId && lhs.vehicleImage == rhs.vehicleImage && lhs.color == rhs.color &&
```
Comment (pauluhn) [inline MojioSDK/Models/Vehicles/VehicleAppearance.swift:70]: #nth multi-line + indentations

File: MojioSDK/Models/Vehicles/VehicleAppearanceStyle.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import Foundation
+
+public enum VehicleAppearanceStyle: String, Codable {
+
+    // Note: values should be compatible with Android's version
+    case sedan = "Sedan"
+    case hatchback = "Hatchback"
+    case smart = "Smart car"
+    case mini = "Mini"
+    case convertable = "Convertible"
```
Comment (pauluhn) [inline MojioSDK/Models/Vehicles/VehicleAppearanceStyle.swift:25]: #? can we fix typo?

**Comment by pauluhn** [comment]
#? is this PR still relevant? the phoenix PR was closed

**Comment by stepanrazin** [comment]
@pauluhn close outdated one to re-create

**Pull request closed without merging**